src/test/java/emissary/output/DropOffUtilTest.java

```diff
@@ -565,14 +571,14 @@ void testGetFileType() {
 
         metadata.clear();
         formsArg = "QUOTED-PRINTABLE";
-        testFileType(bdo, metadata, "TEXT", formsArg);
+        testFileType(bdo, metadata, TEXT, formsArg);
     }
 
     @Test
     void testExtractUniqueFileExtensions() {
         // these should be constants
         final String FILEXT = "FILEXT";
-        final String ORIGINAL_FILENAME = "Original-Filename";
+        final String ORIGINAL_FILENAME = Parameters.ORIGINAL_FILENAME;
```

Comment:
Already statically imported
```suggestion
        final String ORIGINAL_FILENAME = ORIGINAL_FILENAME;
```

Reply:
Yep, you're right - didn't need this anymore.